code/game/gamemodes/wizard/soulstone.dm

```diff
@@ -97,6 +97,10 @@
 	desc = "A wicked machine used by those skilled in magical arts. It is inactive."
 
 /obj/structure/constructshell/attackby(obj/item/O, mob/user, params)
+	if(!iscultist(user) && !iswizard(user))
+		user << "<span class='danger'>An overwhelming feeling of dread comes over you as you pick up the soulstone. It would be wise to be rid of this quickly.</span>"
+		user.Dizzy(120)
```

Comment:
This will trigger on any item hit, not just soulstones.
It also needs to return.

You should put it under line 105;
````
	if(istype(O, /obj/item/device/soulstone))
 		var/obj/item/device/soulstone/SS = O
 		if(!iscultist(user) && !iswizard(user) && !SS.usablility)
			user << "<span class='danger'>An overwhelming feeling of dread comes over you as you attempt to place the soulstone into the shell. It would be wise to be rid of this quickly.</span>"
			user.Dizzy(120)
			return
````